### ISSUE TITLE
The recording of steps does not work when using multithreading inside current test

### ISSUE THREAD

**Issue by Noksa** [Noksa/Allure.NUnit#9]
Using multithreading inside test like `Task.Run(() => AllureLifecycle.Instance.Verify.Pass("Pass"))` throws exceptions

**Versions**
 - Allure: 2.5

**Comment by Noksa** [comment]
Fixed in 2.6

**Comment by Noksa** [comment]
If you use multi-threading inside the test, make sure that before leaving the test, all the threads have done their job of recording the allure steps, otherwise errors may occur.
for example:
```
var task = Task.Run(() =>
            {
                // some code for record allure steps
            });
**task.Wait();**
```

**Comment by Noksa** [comment]
Fixed in 2.6
